src/app/lib/device/chromecast.js

```diff
@@ -24,6 +24,7 @@
         play: function (streamModel) {
             var self = this;
             var subtitle = streamModel.get('subFile');
+            var title = streamModel.get('title').substring(0,50);
```

Comment:
Actually, this number was based on testing using a 29" screen. After 52 chars chromecast started to put a scroll bar which would never go away.